### DISCUSSION TITLE
awesome-pest

### DISCUSSION THREAD

**Discussion by tristan-f-r** [pest-parser/pest#812] [Ideas]
Pest keeps a list of projects using pest in their [GitHub README](https://github.com/pest-parser/.github/blob/bb9e76b7b52094a716f8f83dec9fd498bd13cdc8/profile/README.md).

However, there are other resources for pest that may appear. One notable mention I found was inside [pest's discussions](https://github.com/pest-parser/pest/discussions/791).

Given pest's increasing size, I do think it is appropriate for the organization to have an [awesome](https://github.com/sindresorhus/awesome) repository to keep track of these external resources.

**Comment by tomtau** [comment]
Agreed. I was thinking of writing up a page about "ecosystem tooling" in the book repo, but maybe the dedicated "awesome-pest" repo could be more appropriate?

**Reply by tomtau** [reply]
https://github.com/pest-parser/awesome-pest
